src/plugins/intel_gpu/src/graph/impls/onednn/reduce_onednn.hpp

```diff
@@ -67,6 +67,7 @@ struct ReduceImplementationManager : public ImplementationManager {
         static const std::vector<format::type> supported_formats = {
             format::any,
             format::bfyx,
+            format::byxf,
```

Comment:
could you add test case for planar format in `reduce_gpu_test.cpp`? We already have test cases for onednn, so I think you can just add some instantiation.